### DISCUSSION TITLE
Grid layout responsiveness not working.

### DISCUSSION THREAD

**Discussion by faridsa** [tailwindlabs/tailwindcss#11925] [Help] [answered]
Hello I'm using v3.3.3 to build a contact form with grid layout
My goal is to show one column in small devices and 2 columns in large screens.

So, as shown in this playground [https://play.tailwindcss.com/fRUAjFXQ9b](https://play.tailwindcss.com/fRUAjFXQ9b) I set fieldset in this way `<fieldset class="grid grid-cols-1 gap-4 lg:grid-cols-2 lg:gap-8">` but it remains as 2 columns layout no matter screen size.

Any idea should be much appreciated.

**Comment by nikolailehbrink** [comment, marked as answer]
Hey @faridsa,

The issue you're encountering is due to setting your comments section container with `col-span-2`. For it to function as you intend on smaller screens, you should use `lg:col-span-2`. This ensures that the spanning only operates correctly for the large breakpoint. 

https://play.tailwindcss.com/ivn4zqODxp

**Reply by faridsa** [reply]
Thank you for your accurate reply @nikolailehbrink.
